packages/Python/lldbsuite/test/lang/swift/class_error/main.swift

```diff
@@ -0,0 +1,31 @@
+// Let's try to make sure that frame var prints the dynamic type of a class
+// (and a subclass) conforming to Error correctly (and that we don't crash).
+// <rdar://problem/41025993>
```

Comment:
Let's add the radar number only in the commit message and be sure to include as much explanation in the comment as needed.